src/main/play-doc/operation/Dcos.md

```diff
@@ -0,0 +1,87 @@
+
+Refer to Marathon's documentation for deployment steps given the application definition JSON.
+
+By default ConductR will be deployed with 3 instances. Each ConductR instance will attempt to register itself with a separate framework id, and hence it is expected to have multiple ConductRs appearing in the list of application. These ConductR instances will attempt to form a cluster of its own as part of the startup.
+
+Wait until each of the ConductR instance's health to marked as healthy before proceeding.
+
+If more instances is required, scale up to the desired number of instances once all the 3 instances has been deployed, and ensure each of the ConductR instance's health to marked as healthy before proceeding.
+
+## Deploying bundles
+
+[[ConductR CLI|CLI]] requires the host or ip address of one of the ConductR node in order to communicate
+
+Obtain the host or ip address of one of the ConductR Core nodes that's currently running in the DCOS cluster using the `dcos task` command.
+
+```bash
+Felixs-MBP-2:conductr-doc felixsatyaputra$ dcos task
+NAME                    HOST       USER  STATE  ID
+ConductR                10.0.3.76  root    R    conductr.6e9251af-4e28-11e6-8edd-4219f69bc956
+```
+
+In the example above ConductR has been deployed to `10.0.3.76` - this is the ip address to be obtained. If ConductR has been deployed to multiple nodes, the host address from any these nodes can be selected.
+
+SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment variable with the obtained address.
+
+You are now ready to [[deploy bundles|DeployingBundlesOps]] into ConductR.
+
+## Verifying if bundle has been started successfully
+
+Use the command `conduct info` from the bastion host to verify successful startup of the bundle.
+
+```bash
+ubuntu@ip-10-0-6-165:~$ conduct info
+ID       NAME        #REP  #STR  #RUN
+6e68f05  visualizer     1     0     1
+```
+
+In the example above, a bundle called `visualizer` has been started successfully.
+
+This can also be confirmed by executing the `dcos task` command which will display the bundle running as a task from the context of DC/OS.
+
+```bash
+TODO: example output
+```
+
+The bundle <TODO> task is prefixed with its name followed by its compatibility version. The `ConductR Agent` task belongs to the ConductR Agent process which resides in the same host as where the bundle is running. The ConductR Agent task is responsible for launching bundles on the host where it's started.
+
+
+## Accessing services provided by the bundle
+
+_This section is only relevant to the ConductR 2.0 alpha version. It will be removed once proxying solution provided by ConductR has been consolidated with the solution provided by DC/OS._
+
+Despite the release of ConductR 2.0 alpha version, the work to consolidate existing proxying solution provided by ConductR with the solution provided by DC/OS is still in progress.
```

Comment:
How about: ConductR's automatic gateway proxying solution is not included in the the alpha DC/OS release.